tests/cypress/integration/actions_projects/base_actions_project_task_user.js

```diff
@@ -111,10 +111,39 @@ context('Base actions on the project', () => {
             cy.openTask(taskName.firstTask);
             cy.get('.cvat-constructor-viewer').should('not.exist');
         });
-        it('Logout first user, register second user, logout.', () => {
+        it('Logout first user, register second user, tries to create project and logout.', () => {
             cy.logout();
+            if (Cypress.browser.family !== 'chromium') {
+                cy.get('.cvat-modal-unsupported-platform-warning').within(() => {
+                    cy.contains('button', 'OK').click();
+                });
+            }
             cy.goToRegisterPage();
             cy.userRegistration(firstName, lastName, userName, emailAddr, password);
+            // tries to create project
+            const failProjectName = `${randomString()}`;
+            cy.goToProjectsList();
+            cy.get('#cvat-create-project-button').click();
+            cy.get('#name').type(failProjectName);
+            cy.get('.cvat-constructor-viewer-new-item').click();
+            cy.get('[placeholder="Label name"]').type(labelName);
+            cy.get('.cvat-new-attribute-button').click();
+            cy.get('[placeholder="Name"]').type(attrName);
+            cy.get('.cvat-attribute-type-input').click();
+            cy.get('.cvat-attribute-type-input-text').click();
+            cy.get('[placeholder="Default value"]').type(textDefaultValue);
+            if (multiAttrParams) {
+                cy.updateAttributes(multiAttrParams);
+            }
+            cy.contains('button', 'Done').click();
+            cy.get('.cvat-create-project-content').within(() => {
+                cy.contains('Submit').click();
+            });
+            cy.contains('The project has been created').should('not.exist');
+            cy.get('.cvat-notification-notice-create-project-failed').should('exist');
+            cy.closeNotification('.cvat-notification-notice-create-project-failed');
```

Comment:
``closeNotification()`` also has ``get(.class-name)``. It is why we do not need get before ``closeNotification``. Right?

Reply:
Unnecessary actions removed.